contributors.md

```diff
@@ -269,12 +451,12 @@ contributors:
 
 ## Contributors
 
-Currently, the following people contributed significantly to the software in the
-OSCAR project.
+The following people are actively contributing to the OSCAR project.
 
 <ul>
 {% for p in page.contributors %}
 {% if p.is_active_PI != true %}
+{% if p.is_retired != true %}
```

Comment:
It might be simpler if you just split the list into three, one for active PIs, one for active contributors, one for former contributors. Then you can remove `is_active_PI` and `is_retired` everywhere

Reply:
I believe you have a point.

Reply:
Done.